Assets/Scripts/GraphService.cs

```diff
@@ -77,7 +76,7 @@ public void AddNode(Vector3 position, Quaternion rotation)
         ? Graph.nodes.Max(n => n.id) + 1
         : 0;
 
-      var node = Node.EmptyNode(id, nodeGameObjectFactory.CreateNodeGameObject(position, rotation));
+      var node = Node.EmptyNode(id, nodeGameObjectFactory.CreateDefaultNode(position, rotation));
       node.Position = position;
       node.Rotation = rotation.eulerAngles;
       node.Text.enabled = LabelVisibility;
```

Comment:
{ } initialization syntax would be nice here I think

Reply:
You can only use { } syntax for parameterless constructors, won;t work for static method

Reply:
C#9 will have a nice "with" keyword that will allow the desired syntax but we can't wait for that ;)

Reply:
mb